internal/CMakeLists.txt

```diff
@@ -3,10 +3,14 @@ add_subdirectory(glfw)
 add_subdirectory(glm)
 add_subdirectory(GSL)
 add_subdirectory(jsoncons)
-add_subdirectory(libpng)
 add_subdirectory(oneTBB)
 add_subdirectory(spdlog)
 
+#Ordering this way required as libpng depends on zlib, and freetype depends on both
+add_subdirectory(zlib-ng)
+add_subdirectory(libpng)
+add_subdirectory(freetype)
+
```

Comment:
I'm wondering if this should be after jsoncons? That's how I would've done it if zlib had worked in the first place. Your call though!

Reply:
I moved it - no preference here